packages/react-native/React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -279,6 +279,14 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
     _backedTextInputView.inputAccessoryViewButtonLabel =
         RCTNSStringFromString(newTextInputProps.inputAccessoryViewButtonLabel);
   }
+  
+  NSLog(@"newTextInputProps.disableKeyboardShortcuts: '%d'", newTextInputProps.disableKeyboardShortcuts);
+  NSLog(@"oldTextInputProps.disableKeyboardShortcuts: '%d'", oldTextInputProps.disableKeyboardShortcuts);
```

Comment:
This can be removed

Reply:
Done - 75c39a692587a52a4a9469b7c1994338a5979ed5.